src/app.js

```diff
@@ -84,6 +86,16 @@ function init() {
     })
   );
 
+  app.use(
+    AUTH_STRINGS.staffView.login,
+    ipfilter(
+      process.env.STAFF_VIEW_ALLOWED_IPS
```

Comment:
My guess is we're going to have to also get the IP address from x-forwarded-for header when on Azure. There's probably some reverse proxy that is forwarding the IP address to the backend. It looks like the x-forwarded-for header also includes the port, so you may need to remove that.

Docs: https://www.npmjs.com/package/express-ipfilter (search for "forwarding").

Example: https://stackoverflow.com/questions/58721126/how-can-i-get-the-client-ip-address-in-flask-running-on-an-azure-linux-web-app

Reply:
I *think* I have this working... I'll test on staging after I deploy.

Reply:
Yeah, if it doesn't work in staging, it'll be a pain to debug! We can get the console logs, but it's a pain since our staging logs and prod logs are mixed together.